app/src/main/java/com/mapbox/maps/testapp/examples/style/TileJsonActivity.kt

```diff
@@ -0,0 +1,74 @@
+package com.mapbox.maps.testapp.examples.style
+
+import android.os.Bundle
+import android.util.Log
```

Comment:
Should use common logger instead.

Reply:
I can suggest to write own detekt or ktlint rule for checking this

Reply:
Thinking from a customer's perspective, I think we should use `android.util.Log` in the example app, and use common logger in our internal logging in sdk and plugin modules.

Reply:
For now, I'm ok with Log or Logger. I'm not seeing a need for the one or other in the testapp module.
Long term, when we won't emit logs of the common SDK lower as error by default. I would suggest using Log else a user copy/pastes code and it wouldn't see any logs.